### ISSUE TITLE
Trouble recreating the relighting result following training instructions

### ISSUE THREAD

**Issue by kschen405** [xiao7199/Latent_Intrinsics#2]
Hello, thanks for the amazing work! 

However, I couldn't recreate the relighting results from your pretrained model. I trained the model following the setting from srun.sh on a single GPU using gradient accumulatation. 

Below are the relighting result (by saving relight_img2 in eval_relight() ) using the pretrained model.

![Image](https://github.com/user-attachments/assets/bc11eae2-0eaa-4643-af6d-fc81c680c9ea)

Below are the relighting result (by saving relight_img2 in eval_relight() ) following the training instructions.

![Image](https://github.com/user-attachments/assets/86d45abd-6ca7-4cf1-9b3a-76db9b9c090b)

 
Also, I noticed that the pretrained model seems to be in its 140 epoch, while the number of epochs in srun.sh is 120 epoch.

**Comment by xiao7199** [comment]
Hello, thank you for your interest in our work.
We haven't tested this on the single-GPU setup. When do you accumulate grads? Have you tried dividing the loss by the number of accumulation runs? 

We trained the model to 140 epochs, but found that the performance saturates at around 120 epochs.
